modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -191,6 +191,16 @@ public void flatten(Swagger swagger) {
         }
     }
 
+    private void fixStringModel(ModelImpl m) {
+        if (m.getType() != null && m.getType().equals("string") && m.getExample() != null) {
+            String example = m.getExample().toString();
+            if (example.substring(0, 2).equals("\"") &&
+                    example.substring(example.length() - 2).equals("\"")) {
+                m.setExample(example.substring(2, example.length() - 2));
```

Comment:
I've test it and it works. And we want to cut "\"", which is 2 chars.

For example:
```
0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8
\ | " | H | E | L | L | O | \ | "
```

First condition will return` \"` which is true
Second condition will return 9 - 2 = 7 -> `\"`
Then we simply remove everything between both which returns `HELLO`

I've test it and it works :) Let me know if it doesn't work in your case and I'll look into it!